Tasks/DeployAzureResourceGroup/Environment.ts

```diff
@@ -192,7 +191,7 @@ export class RegisterEnvironment {
             var fqdns = {}
             for (var i = 0; i < publicAddresses.length; i++) {
                 var publicAddress = publicAddresses[i];
-                var publicAddressId = publicAddress["id"];
+                var publicAddressId = publicAddress["id"]; //Didn't convert to lowercase
```

Comment:
It's an identifier used globally, conversion to lowercase is not necessary I guess